File: Language_Coder/src/array/Code570.java
package array;

public class Code570 {

	public static void main(String[] args) {
		int[][] arr = { { 1, 1, 1, 1, 1 }, { 1, 2, 3, 4, 5 }, { 1, 3, 6, 10, 15 }, { 1, 4, 10, 20, 35 },
				{ 1, 5, 15, 35, 70 } };

		for (int i = 0; i < 5; i++) {
			for (int j = 0; j < 5; j++) {
				System.out.print(arr[i][j]);
				if (j != 4)
					System.out.print(" ");
			}
			System.out.println();
		}
	}

}

File: Language_Coder_Cpp/Code570/Code570.cpp
// Code570.cpp

#include <iostream>

using namespace std;

int main()
{
	int** arr = new int* [5];

	for (int i = 0; i < 5; i++) {
		arr[i] = new int[5];
	}

	for (int i = 0; i < 5; i++) {
		arr[0][i] = 1;
		arr[i][0] = 1;
	}

	for (int r = 1; r < 5; r++) {
		for (int c = 1; c < 5; c++) {
			arr[r][c] = arr[r - 1][c] + arr[r][c - 1];
		}
	}

	for (int r = 0; r < 5; r++) {
		for (int c = 0; c < 5; c++) {
			cout << arr[r][c] << " ";
		}
		cout << endl;
	}
}